controllers/components/feastoperator/feastoperator_support.go

```diff
@@ -18,7 +18,7 @@ const (
 var (
 	imageParamMap = map[string]string{
 		"RELATED_IMAGE_FEAST_OPERATOR": "RELATED_IMAGE_ODH_FEAST_OPERATOR_IMAGE",
-		"RELATED_IMAGE_FEATURE_SERVER": "RELATED_IMAGE_ODH_FEAST_FEATURE_SERVER_IMAGE",
+		"RELATED_IMAGE_FEATURE_SERVER": "RELATED_IMAGE_ODH_FEAST_SERVER_IMAGE",
 	}
```

Comment:
@zdtsw  can you confirm this name again the comments on Jira says `RELATED_IMAGE_ODH_FEATURE_SERVER_IMAGE`

Reply:
cc @riprasad